### PR TITLE
Replace lengths of stay with days

### LINKED ISSUE #337: ["bug"] Fix misleading LOS parameter names.
The fields

- Hospital Length of Stay
- ICU Length of Stay
- Vent Length of Stay

These fields can imply that the user should entire the average hospital length of stay for a patient that required ICU.  And similarly for Vent.  Instead, they specifically mean the time in the ICU, and the time on the vent.

### Description:
The fields should be updated.  

This should say
- Hospital Length of Stay
- Days in ICU
- Days on Ventilator

### PR DESCRIPTION
**Pull request by jlubken**
Clusters all days widgets together:
- Infectious Days
- Hospital Length of Stay (days)
- Days in ICU
- Days on Ventilator

Then the percent widgets afterward.

Closes #337

### CODE REVIEW

File: src/penn_chime/cli.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def parse_args():
             "Doubling time before social distancing (days)",
             True,
         ),
-        ("--hospitalized-los", int, 0, None, "Hospitalized Length of Stay (days)", True),
+        ("--hospitalized-days", int, 0, None, "Hospital Length of Stay (days)", True),
```
Comment (mdbecker) [inline src/penn_chime/cli.py]: Probably this should read something like `Average Hospital Length of Stay` just to be clear (or `Mean`?)

File: src/penn_chime/cli.py [hunk 2]
```diff
@@ -83,7 +83,7 @@ def parse_args():
             "Hospitalized Rate: 0.00001 - 1.0",
             True,
         ),
-        ("--icu-los", int, 0, None, "ICU Length of Stay (days)", True),
+        ("--icu-days", int, 0, None, "Days in ICU", True),
```
Comment (mdbecker) [inline src/penn_chime/cli.py]: Similar to above probably we should specify this is the average

File: src/penn_chime/cli.py [hunk 3]
```diff
@@ -104,7 +104,7 @@ def parse_args():
             True,
         ),
         ("--population", int, 1, None, "Regional Population >= 1", True),
-        ("--ventilated-los", int, 0, None, "Ventilated Length of Stay (days)", True),
+        ("--ventilated-days", int, 0, None, "Days on Ventilator", True),
```
Comment (mdbecker) [inline src/penn_chime/cli.py]: Similar to above probably we should specify this is the average

**Review by mdbecker — CHANGES_REQUESTED** [review]

File: src/penn_chime/presentation.py [hunk 4]
```diff
@@ -196,27 +196,27 @@ def display_sidebar(st, d: Parameters) -> Parameters:
         "Ventilated %(total infections)",
         value=d.ventilated.rate,
     )
-    hospitalized_los_input = NumberInput(
+    hospitalized_days_input = NumberInput(
         st_obj,
         "Hospital Length of Stay",
```
Comment (ecsmyth) [inline src/penn_chime/presentation.py]:
```suggestion
        "Average Hospital Length of Stay (days)",
```

File: src/penn_chime/presentation.py [hunk 5]
```diff
@@ -196,27 +196,27 @@ def display_sidebar(st, d: Parameters) -> Parameters:
...
-        value=d.hospitalized.length_of_stay,
+        value=d.hospitalized.days,
         step=1,
         format="%i",
     )
-    icu_los_input = NumberInput(
+    icu_days_input = NumberInput(
         st_obj,
-        "ICU Length of Stay",
+        "Days in ICU",
```
Comment (ecsmyth) [inline src/penn_chime/presentation.py]:
```suggestion
        "Average Days in ICU",
```

File: src/penn_chime/presentation.py [hunk 6]
```diff
@@ -196,27 +196,27 @@ def display_sidebar(st, d: Parameters) -> Parameters:
...
-        value=d.icu.length_of_stay,
+        value=d.icu.days,
         step=1,
         format="%i",
     )
-    ventilated_los_input = NumberInput(
+    ventilated_days_input = NumberInput(
         st_obj,
-        "Vent Length of Stay",
+        "Days on Ventilator",
```
Comment (ecsmyth) [inline src/penn_chime/presentation.py]:
```suggestion
        "Average Days on Ventilator",
```

File: src/penn_chime/parameters.py [hunk 7]
```diff
@@ -11,7 +11,7 @@
 
 # (0.02, 7) is 2%, 7 days
 # be sure to multiply by 100 when using as a default to the percent widgets!
-RateLos = namedtuple("RateLos", ("rate", "length_of_stay"))
+RateDays = namedtuple("RateDays", ("rate", "days"))
```
Comment (ecsmyth) [inline src/penn_chime/parameters.py:14]:
```suggestion
DaysAtRate= namedtuple("PeriodAtRate", ("rate", "days"))
```
Not necessary for this PR but I find the name confusing.

**Review by ecsmyth — CHANGES_REQUESTED** [review]
Few strings in `presentation.py` that need to be brought to parity with `cli.py`. I also see four instances of `RateLos` in `sidebar.py`

File: src/penn_chime/parameters.py (hunk 7 above)
Comment (jlubken) [reply src/penn_chime/parameters.py:14]: This is actual a tuple of (rate of disposition within the infected) and (average days of disposition), so the rate is something else.

**Pull request merged**